src/ucs/profile/profile.c

```diff
@@ -76,13 +31,16 @@ const char *ucs_profile_mode_names[] = {
     [UCS_PROFILE_MODE_LAST]  = NULL
 };
 
-static ucs_profile_global_context_t ucs_profile_global_ctx = {
-    .locations     = NULL,
-    .num_locations = 0,
-    .max_locations = 0,
-    .mutex         = PTHREAD_MUTEX_INITIALIZER,
-    .thread_list   = UCS_LIST_INITIALIZER(&ucs_profile_global_ctx.thread_list,
-                                          &ucs_profile_global_ctx.thread_list),
+ucs_profile_global_context_t ucs_profile_global_ctx = {
```

Comment:
1. maybe rename it to ucs_profile_deafault_ctx ?
2. rename the type ucs_profile_global_context_t to ucs_profile_context_t (since it's not global anymore, can come from ucc)